src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java

```diff
@@ -200,6 +321,10 @@ public FilePath getPowerShellWrapperFile(FilePath ws) throws IOException, Interr
             return controlDir(ws).child("powershellWrapper.ps1");
         }
 
+        public FilePath getTemporaryOutputFile(FilePath ws) throws IOException, InterruptedException {
+            return controlDir(ws).child("temporaryOutput.txt");
```

Comment:
This gets cleaned up automatically when the control directory is cleaned up, right? Also, the directory name is randomized, and there would never be more than one `temporaryOutput.txt` file per step, right?

Reply:
that is correct, the powershell script pipes the output directly to `temporaryOutput.txt` which is then converted to the final `jenkins-output.txt`. There will only be one `temporaryOutput.txt` file per script and it lives with the other files in the control directory so it gets cleaned up at the same time.

Reply:
Funny aside: a lot of this code existed before the existence of `powershellhelper.ps1`. I ended up re-implementing a lot of that old code unintentionally.